docs/docs/user/migration-guide/v2-v3.md

```diff
@@ -63,3 +63,17 @@ steps:
       command: deploy --execute
       # ...
 ```
+
+## [Allow overriding config files](https://github.com/bakdata/kpops/pull/391)
+
+`environment` is no longer specified in `config.yaml`. Instead, it can be either set via the cli flag `--environment` or with the environment variable `KPOPS_ENVIRONMENT`.
```

Comment:
Please mention that the `--config` is now pointing at the directory and not the file anymore

Reply:
Added some text, lmk what you think